File: ex1/js/main.js
console.log("connected");

maxNumber = 10
// listArray = []
// for (var i = 1; i <= maxNumber; i++) {
//   listArray.push(i)
// }
// answerArray = []
// answer = 0
//
// for (var i = 0; i < listArray.length; i++) {
//   if ((listArray[i] % 3 === 0) || (listArray[i] % 5 === 0))
//   answerArray.push(listArray[i])
// }
//
// for (var i = 0; i < answerArray.length; i++) {
//   answer = answer + answerArray[i]
// }
//
// console.log(answer);

/////////

let listArray = _.range(maxNumber);
let answerArray = _.filter( listArray, function (x) {
  return x % 3 === 0 || x % 5 === 0
})
answer = 0
_.each(answerArray, function (x) {
  answer = answer + x
})
console.log(answerArray);
console.log(answer);

File: ex1/main.rb
# If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
#
# Find the sum of all the multiples of 3 or 5 below 1000.

max_num = 999
test_array = (1..max_num)
answer_array = []

test_array.each do |x|
  if x % 3 == 0 || x % 5 == 0
    answer_array.push(x)
  end
end

answer = answer_array.sum
puts answer
